extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -59,12 +59,11 @@ import {getContextMetadata} from '../../../src/iframe-attributes';
 // import '../../amp-ad/0.1/amp-ad-ui';
 // import '../../amp-ad/0.1/amp-ad-xorigin-iframe-handler';
 
-/** @type {string} */
-const METADATA_STRING = '<script type="application/json" amp-ad-metadata>';
-
-/** @type {string} */
-const METADATA_STRING_NO_QUOTES =
-      '<script type=application/json amp-ad-metadata>';
+/** @type {Array<string>} */
+const METADATA_STRINGS = [
+  '<script amp-ad-metadata type=application/json>',
+  '<script type="application/json" amp-ad-metadata>',
+  '<script type=application/json amp-ad-metadata>'];
```

Comment:
I'd prefer to be backwards compatible for now but as mentioned in the description:  "Add support for multiple versions (to be cleaned up in the future once stable)."